scripts/us_epa/air_emissions_inventory/download.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import os
+
+input_directory = r'scripts/us_epa/air_emissions_inventory/input_html_files/'
+os.mkdir("scripts/us_epa/air_emissions_inventory/input_files")
+output_directory = r'scripts/us_epa/air_emissions_inventory/input_files/'
+for filename in os.listdir(input_directory):
```

Comment:
Use temp directory (tempfile package) to download html files; once it is downloaded in same module convert html to csv.